widget.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <version>0.0.1</version>
+        <id>SampleWidget</id>
+        <authors>SampleDev</authors>
+        <projectUrl>https://github.com/UiPath/assistant-plugin-creator</projectUrl>
+		    <description>A widget for UiPath Assistant</description>
```

Comment:
I've noticed in my IDE that here we have two tabs at the beginning instead of spaces.
![image](https://user-images.githubusercontent.com/24717857/105326380-5b6e0d00-5bd6-11eb-8df9-c659d37feb18.png)